test/fixtures/es-module-loaders/string-sources.mjs

```diff
@@ -22,7 +22,11 @@ const SOURCES = {
 }
 export function resolve(specifier, context, next) {
   if (specifier.startsWith('test:')) {
-    return { url: specifier, importAssertions: context.importAssertions };
+    return {
+      importAssertions: context.importAssertions,
+      shortCircuit: true,
```

Comment:
So it seems like this PR introduces a breaking change for current loaders, in that many of them will need to be updated to add `shortCircuit: true`. I think this is fine, and I think we already have a good, explicit error guiding people to update this, but we might want to call it out somewhere. Unfortunately we don’t have a changelog or something similar within the ESM docs, so maybe we don’t have a place to mention it . . . ? cc @aduh95

Reply:
There's the history table of the Loaders section: https://github.com/nodejs/node/blob/517fe9584aa85b5cba1f90eff103d30c4b2a5113/doc/api/esm.md#L677-L681

Reply:
added